### ISSUE TITLE
Adaptive crosshair sucks!

### ISSUE THREAD

**Issue by Psychodeus** [Exopandora/ShoulderSurfing#92]
![image](https://user-images.githubusercontent.com/93028949/194406613-3bbdd722-e1b3-4628-bb29-de28aa3ccc7c.png)
![image](https://user-images.githubusercontent.com/93028949/194406672-f640c36d-f1e3-4dde-ab1e-6e665b39c4b3.png)
Video: https://imgur.com/a/xFckpD2

Those two sucks, let alone static. The only reliable option is **static with first person.**
If you ever play Apex legends (I know the comparassion is far from minecraft, but bear with me) the crosshair in 3rd person mode will always land the projectile there, in the middle.

But, this Shoulder Surfing is not even land the projectile on both static or adaptive crosshair mode.

Please, if you take this as feedback, let the "main" crosshair alone. Focus on the "Shadow crosshair" the function of that shadow crosshair is to show where it lands 
![image](https://user-images.githubusercontent.com/93028949/194410139-e1075f46-3637-4fe5-a32b-959b2e4021dc.png)
(and please twist it so the shadow crosshair below the main crosshair, not around it).

![image](https://user-images.githubusercontent.com/93028949/194409253-343162b2-089d-4405-b32b-e4d632b47bb3.png)
Green: Main crosshair.
Blue: Shadow crosshair.

So we got 2 crosshair on screen, one faded (Blue - Shadow), and one still default (Green - Main, Static).
Even better if we could see the projectile path. Thanks.

**Comment by Exopandora** [comment]
Are you using a different render resolution? The bow is supposed to shoot where the crosshair is aiming at (but hits a little lower).

https://user-images.githubusercontent.com/18708736/194416813-466d95d8-33be-488e-93ed-a4dbd9776c47.mp4

**Comment by Psychodeus** [comment]
![image](https://user-images.githubusercontent.com/93028949/194421401-1d0b75de-cc46-44cf-98db-192a0c4ebb01.png)
I set it as this size, but it's just the scale when it's windowed, I switch oftenly with borderless and window fullscreen.

Should I set in the launcher as 1920 x 1080?

**Comment by Psychodeus** [comment]
Oh, and about your video, try farther.

**Comment by Psychodeus** [comment]
I already set to 1920 x 1080 at launcher settings, and this is still the result https://imgur.com/a/O3GBc3K

**Comment by Exopandora** [comment]
> Oh, and about your video, try farther.
The video is just a demonstration and also applies to long ranges. The video you have shown clearly has a bug. I suggest you try SSR without any other mods (and perhaps vanilla launcher) and get a feeling of how its supposed to work.
Note: Some shaders render the game at a higher resolution which may cause issues with SSR.

**Comment by Psychodeus** [comment]
I don't know what SSR is, only know USSR (lol).
I'm using TLauncher, and using ~70 mods.

**Comment by Exopandora** [comment]
SSR: **S**houlder **S**urfing **R**eloaded

**Comment by Psychodeus** [comment]
Thanks will try. Adios~

**Comment by Psychodeus** [comment]
Still need to fix the bugs right?

**Comment by Exopandora** [comment]
If its a compatibility bug that can be fixed, sure.

**Comment by Exopandora** [comment]
It would be cool if you could figure out what other mods cause the issue or if its general issue for you (this is the case when the issue is present when no other mods are installed). To quickly find the mods you can use [binary search](https://gist.github.com/TomeWyrm/a8cff19ce112edbc3c8a). If the issue is caused by a shader pack, please provide the name of the shader pack and the settings you use.

**Comment by Psychodeus** [comment]
Yeah well before finding that, I need to find my mood and time 🙆‍♂️
